client-web/src/exlibris-app/exlibris-book-tile.html

```diff
@@ -17,22 +17,23 @@
   ~ (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
   ~ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
   -->
-
 <link rel="import" href="../../bower_components/polymer/polymer-element.html">
 
 <link rel="import" href="../../bower_components/iron-flex-layout/iron-flex-layout.html">
 <link rel="import" href="../../bower_components/iron-image/iron-image.html">
 <link rel="import" href="../../bower_components/iron-scroll-threshold/iron-scroll-threshold.html">
 <link rel="import" href="../../bower_components/iron-list/iron-list.html">
+<link rel="import" href="../../bower_components/polymer/lib/elements/dom-if.html">
+<link rel="import" href="./book_values.html">
 
-<dom-module id="book-grid">
+<dom-module id="exlibris-book-tile">
     <template>
         <style>
             :host {
                 display: block;
             }
 
-            .bookContent {
+            .book-content {
```

Comment:
Why do we use `book-content` if we have `host`?

Reply:
`margin: 20px;` doesn't work in `host`